highcharts-angular/package.json

```diff
@@ -21,7 +21,7 @@
   "peerDependencies": {
     "@angular/common": ">=19.0.0",
     "@angular/core": ">=19.0.0",
-    "highcharts": ">=9.0.0"
+    "highcharts": ">=12.0.0"
```

Comment:
Since v12 [modules no longer require initialization](https://github.com/highcharts/highcharts/blob/master/changelog/highcharts/12.0.0.md#upgrade-notes). That is a breaking change for us. Supporting only v12 allow us to simplify our code. We no longer need to identify the version and conditionally call `module.default(Highcharts)`.

And since it's been historically relatively painless to upgrade Highcharts, I don't see why users wouldn't upgrade at the same time as they upgrade highcharts-angular. It is a rather low-effort for them, and avoid us supporting old things.

I am not able to find an official old version support policy. So in the absence of such, I can only assume that only the latest version of Highchart is supported. That would mean v11 is EOL.

@karolkolodziej, are you able to share something regarding which versions of Highcharts should be supported ?